rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java

```diff
@@ -136,10 +136,11 @@ public void perform(GraphRewrite event, EvaluationContext context, XmlFileModel
             if (StringUtils.isNotBlank(ejbName))
             {
                 LOG.info("Looking up name: " + ejbName);
-                for (EjbSessionBeanModel ejb : ejbSessionBeanService.findAllByProperty(EjbMessageDrivenModel.EJB_BEAN_NAME, ejbName))
+                for (EjbSessionBeanModel ejb : ejbSessionBeanService.findAllByProperty(EjbSessionBeanModel.EJB_BEAN_NAME, ejbName))
```

Comment:
This was a mistake I guess.

Reply:
Arguably, it should actually be org.jboss.windup.rules.apps.javaee.model.EjbBeanBaseModel#EJB_BEAN_NAME.